modules/network-observability-deploy-network-policy.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/network-observability-network-policy.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-deploy-network-policy_{context}"]
+= Configuring an ingress network policy by using the FlowCollector custom resource
+You can configure the `FlowCollector` custom resource to deploy an ingress Network Policy for Network Observability by setting the `spec.NetworkPolicy.enable` specification to `true`. By default, the specification is `false`.
+
+Ensure that Network Observability components you have configured can communicate with components that are external to the namespace, which might include:
+
+   * Connection to Loki (as defined in `FlowCollector` `spec.loki`)
+   * Connection to Kafka (as defined in `FlowCollector` `spec.kafka`)
+   * Connection to any exporter (as defined in FlowCollector `spec.exporters`)
```

Comment:
Is this right? It seems like we need more here. 
```suggestion
   * Connection to Loki (as defined in the `FlowCollector` CR `spec.loki` parameter)
   * Connection to Kafka (as defined in the `FlowCollector` CR `spec.kafka` parameter)
   * Connection to any exporter (as defined the `FlowCollector` CR `spec.exporters` parameter)
```
Will people know what this means or should we provide examples?

Reply:
I think NetObserv users should be pretty familiar with the different storage/connection options, i.e. Loki and Kafka. We have topics about Exporters and also configuring Loki and Kafka are part of the initial installation of the Operator. Maybe its worth a link over there in case there is a question.